hw03_masone2.py

```diff
@@ -0,0 +1,247 @@
+######################################################################
+# Author: Beth Mason
+# Username: Masone2
+# Assignment: HW03: Fully Functional Gitty Psychedelic Robotic Turtles
+# Purpose: To demonstrate the turtle library, funcshons, and git
+# google doc link: https://docs.google.com/document/d/1uEDcJfrSPKVqRZIrBYCS930wwOhCPhb6U_Q640EEbRM/edit?usp=sharing
+######################################################################
+# Acknowledgements:
+#HW02: Loopy Turtles, Loopy Languages and its assosiated acknowledgments
+######################################################################
+import turtle
+import random
+import math
+
+def full_head(catpill):
+    """makes the head of the full catipiller"""
+    catpill.goto(0, 70)
+    catpill.left(135)
+    catpill.fillcolor("crimson")
+    catpill.begin_fill()
+    catpill.circle(45, 170)
+    catpill.end_fill()
+def full_mouth(catpill):
```

Comment:
PEP8 wants two blank lines before each function definition. I am going to let it slide this time though, because the rest of this code looks great. Good job!